lua/nvim-treesitter/install.lua

```diff
@@ -64,8 +64,10 @@ local function get_parser_install_info(lang, validate)
 end
 
 local function load_lockfile()
-  local filename = utils.join_path(utils.get_package_path(), "lockfile.json")
-  lockfile = vim.fn.filereadable(filename) == 1 and vim.fn.json_decode(vim.fn.readfile(filename)) or {}
+  if #lockfile == 0 then
```

Comment:
the lockfile needs to be loaded multiple times in order to make hot-reloading on `:TSUpdate` working

Reply:
Oh right

Reply:
@vigoux Maybe I was wrong and I'm explicitly setting `lockfile = {}` on reset. Then, this would not be necessary